pyaedt/emit_core/__init__.py

```diff
@@ -32,3 +62,4 @@ def _set_api(aedt_version):
         global EMIT_API_PYTHON
         EMIT_API_PYTHON = import_module("EmitApiPython")
         logger.info("Loaded {}".format(EMIT_API_PYTHON.EmitApi().get_version(True)))
+    _init_enums()
```

Comment:
@myoung301 Don't the enums require v232 or newer? This looks like its outside the `if`

Reply:
@jsalant22 I applied the change.